frontend/apps/core-app/src/components/SubFormTable/SubFormTable.component.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import { TableInstance, Row, Cell } from 'react-table';
+import { Box, HStack, VStack, Text, Button, ScrollView } from 'native-base';
+import { Icon } from 'core-design-system';
+
+type Props = {
+  table: TableInstance<any>;
+  onDelete?: (idx: number) => void;
+};
+
+export const SubFormTableComponent: React.FC<Props> = ({ table, onDelete }) => {
+  const handleDelete = (idx: number) => () => onDelete?.(idx);
+
+  return (
+    <ScrollView width="100%" horizontal>
+      <VStack width="100%">
+        <HStack width="100%">
+          {table.columns.map((col) => (
+            <Box
+              key={col.id}
+              width={col.width}
+              p="2"
+              borderBottomColor="neutral.300"
```

Comment:
not a big issue, and design might change anyway, but the bottom border on the table header is shorter due to the delete buttons, looks weird